crypto/evp/evp_local.h

```diff
@@ -27,8 +25,11 @@ struct evp_md_ctx_st {
     /* Update function: usually copied from EVP_MD */
     int (*update) (EVP_MD_CTX *ctx, const void *data, size_t count);
 
-    /* Provider ctx */
-    void *provctx;
+    /*
+     * Opaque ctx returned from a providers digest algorithm implementation
+     * OSSL_FUNC_digest_newctx()
+     */
```

Comment:
This comment seems a bit bloated at this point.  How about `/* Algorithm-specific ctx */`?

Reply:
I prefer the longer comment.

Reply:
So why only here and not *everywhere where we have the algctx field?

Reply:
Change to make them all more verbose noted.

I find it amazing, though, that extended verbosity is chosen for this item specifically.  I kinda understand it considering that there seems to be a lot of confusion...  but in a year, will we still feel the same?

Reply:
Anyway, I'm not going to block on terms of verbosity, just wanted to express my surprise over this choice, *this time*

Reply:
This may be trying to be kind to others trying to understand what it is.